newdle/schemas.py

```diff
@@ -19,3 +21,30 @@ def sort_users(self, data, many, **kwargs):
         if many:
             data = sorted(data, key=lambda x: x['name'].lower())
         return data
+
+
+class ParticipantSchema(mm.Schema):
+    name = fields.String(required=True)
+    email = fields.String()
+    auth_uid = fields.String()
+
+
+class SlotSchema(mm.Schema):
+    start = fields.DateTime(required=True, format=DATETIME_FORMAT)
+    end = fields.DateTime(required=True, format=DATETIME_FORMAT)
+
+
+class NewNewdleSchema(mm.Schema):
+    title = fields.String(validate=lambda x: len(x) >= 3, required=True)
+    duration = fields.Int(required=True, validate=lambda x: x % 15 == 0)
+    timezone = fields.String(
+        validate=lambda x: x in common_timezones_set, required=True
```

Comment:
`validate=common_timezones_set.__contains__`? :D (maybe a bit too magic though)

Reply:
argh. no. I really don't see a problem with using lambda.

Reply:
(it's also more readable)

Reply:
yeah, i didn't expect anyone to change this one :D